File: docs/how-tos/rotate-key/rust/src/step3.rs
// PART 2
println!("9. Generating new Verkey of Trust Anchor in the wallet");
let trustee_temp_verkey = did::replace_keys_start(wallet_handle, &trustee_did, &"{}").wait().unwrap();

println!("10. Building NYM request to update new verkey to ledger");
let replace_key_nym_request: String = ledger::build_nym_request(&trustee_did, &trustee_did, Some(&trustee_temp_verkey), None, Some("TRUST_ANCHOR")).wait().unwrap();

println!("11. Sending NYM request to the ledger");
let _replace_key_nym_sign_submit_result: String = ledger::sign_and_submit_request(pool_handle, wallet_handle, &trustee_did, &replace_key_nym_request).wait().unwrap();

println!("12. Applying new Trust Anchor's Verkey in wallet");
did::replace_keys_apply(wallet_handle, &trustee_did).wait().unwrap();

File: docs/how-tos/rotate-key/nodejs/step3.js
    // 9.
    log('9. Generating new verkey of trust anchor in wallet')
    const newVerkey = await indy.replaceKeysStart(walletHandle, trustAnchorDid, "{}")
    logValue('New Trust Anchor Verkey: ', newVerkey)

    // 10.
    log('10. Building NYM request to update new verkey to ledger')
    const nymRequestForNewVerkey = await indy.buildNymRequest(trustAnchorDid, trustAnchorDid, newVerkey, undefined, 'TRUST_ANCHOR')

    // 11.
    log('11. Sending NYM request to the ledger')
    await indy.signAndSubmitRequest(poolHandle, walletHandle, trustAnchorDid, nymRequestForNewVerkey)

    // 12.
    log('12. Apply new verkey in wallet')
    await indy.replaceKeysApply(walletHandle, trustAnchorDid)

